frontend/src/app/main/data/comments.cljs

```diff
@@ -86,9 +86,9 @@
   (us/assert ::comment-thread thread)
   (ptk/reify ::update-comment-thread-status
     ptk/WatchEvent
-    (watch [_ _ _]
+    (watch [_ state _]
       (let [done #(d/update-in-when % [:comment-threads id] assoc :count-unread-comments 0)]
-        (->> (rp/mutation :update-comment-thread-status {:id id})
+        (->> (rp/mutation :update-comment-thread-status {:id id :share-id (get-in state [:viewer-local :share-id])})
```

Comment:
The same here, the share-id can be extraced using `->` on the let bindings.

Reply:
done